hutch_python/happi.py

```diff
@@ -118,6 +140,17 @@ def get_happi_objs(
         if device.name in exclude_devices:
             containers.remove(device)
 
+    # Load additional devices
+    for device_dict in additional_devices:
+        for key, value_list in device_dict.items():
+            for value in value_list:
+                reqs = {key: value}
+                results = client.search(**reqs)
+                add_devices = [
+                    res.item for res in results if res.item.name not in dev_names]
```

Comment:
`dev_names` is not guaranteed to be bound here, it's only created in one of the `else` clauses above.